lib/cmis/query_result.rb

```diff
@@ -3,11 +3,13 @@ class QueryResult
     attr_reader :results
     attr_reader :num_items
     attr_reader :has_more_items
+    attr_reader :debug_info
 
-    def initialize(results, num_items, has_more_items)
+    def initialize(results, num_items, has_more_items, debug_info)
```

Comment:
On second thought, can you make **nil** the default argument, as QueryResult is instantiated in two other places too.